core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -190,7 +179,7 @@ func (ec *EthConfirmer) Close() error {
 	return ec.StopOnce("EthConfirmer", func() error {
 		ec.ctxCancel()
 		ec.wg.Wait()
-
+		ec.orm.Close()
```

Comment:
Be careful! You passed the ORM in as an argument - are you sure it isn't shared? I would suggest either making one shared ORM for the whole tx manager lifecycle, or instantiating it in every instance where it is needed.

Reply:
good catch. will do 👍

Reply:
I think I'll create one shared ORM for the whole lifecycle and close it once from the top-level

Reply:
Actually, it seems like we may start multiple eth confirmers during the life cycle of 1 Txm so I think it's best to create one orm per component then. (ref: https://github.com/smartcontractkit/chainlink/blob/augustus.BCI-268.p4-ethconfirmer/core/chains/evm/txmgr/txmgr.go#L227-L229) 

Otherwise, if we had one shared you couldn't close the ORM when restart the eth confirmer. Also, by having seperate ORMs we can tie each orm to the context of component at a future point in time

Reply:
I commented earlier too, but yes, follow a consistent pattern of where is orm created, and where it is closed. The component closing the ORM, is the owner of it.
If you decide to share the ORM, then the TxMgr should be the only one closing it at the end.